packages/langium/src/grammar/langium-grammar.langium

```diff
@@ -1,8 +1,9 @@
-/******************************************************************************
- * Copyright 2021 TypeFox GmbH
- * This program and the accompanying materials are made available under the
- * terms of the MIT License, which is available in the project root.
- ******************************************************************************/
+// ******************************************************************************
```

Comment:
I guess this comment formatting is changed to be not recognized as JSdoc, right?

Reply:
Correct 👍